### DISCUSSION TITLE
Curation report - Week 3 2022

### DISCUSSION THREAD

**Discussion by nhungpham1707** [wikipathways/GPML-Repo#39] [Curation reports]
Hi admins,

Please find below my curation report.

- Last week there were 5 recently changed pathways, 6 under construction, 5 stub pathways and 3 need approval tags. I already approved wherever needed.

- There are 3 incomplete pathways that need actions including WP5078, WP5080 (Martina) and WP5091 (Marvin). I already contacted the authors about it.

- A general comment, there are many pathways that look very nice but the node labels are hardly readable (e.g. WP5130, WP5154, WP5019, WP5150, and WP5095). This is such a pity since it will be inconvenient to use them for publication. 

Next up: Fin

Regards,
Nhung

**Comment by khanspers** [comment]
Thanks Nhung! For point #3 about the pathways that are hard to read, at least some of these can be fixed by just increasing the node label font, node size or updating the overall dimensions of the pathway. I might have some time later this week to do one or two of these.
